### DISCUSSION TITLE
can i use this for tp link wn823n adapter?

### DISCUSSION THREAD

**Discussion by mohammad-zadeed** [Mange/rtl8192eu-linux-driver#316] [Q&A] [answered]
i am using tp link wn823n in ubuntu 22.04 but the out of the box drivers are very slow and sometimes my internet stops and i have to reconnect it 
if i use this driver , will it solve my problem?

**Comment by CGarces** [comment, marked as answer]
wn823n should work on with this repository. I don't know if will work better that your current  driver (maybe mainline kernel?)
